strr-api/src/strr_api/schemas/schemas/host-registration.json

```diff
@@ -348,13 +247,32 @@
           "documentType"
         ]
       }
+    },
+    "strRequirements": {
+      "type": "object",
+      "properties": {
+        "isBusinessLicenceRequired": {
+          "type": ["boolean", "null"]
+        },
+        "isPrincipalResidenceRequired": {
+          "type": ["boolean", "null"]
+        },
+        "isStrProhibited": {
+          "type": ["boolean", "null"]
+        },
+        "isStraaExempt": {
+          "type": ["boolean", "null"]
+        },
+        "organizationNm": {
+          "type": ["string", "null"]
+        }
+      }
     }
   },
   "required": [
     "primaryContact",
     "unitAddress",
     "unitDetails",
-    "listingDetails",
-    "principalResidence"
+    "listingDetails"
```

Comment:
Might as well remove 'listingDetails' too I think. It's being set as an empty list via the UI